--- origsrc/scons-0.98.5/setup.py	2008-06-10 00:21:08.000000000 -0500
+++ src/scons-0.98.5/setup.py	2008-06-17 23:28:59.218750000 -0500
@@ -353,7 +353,7 @@
             if is_win32:
                 dir = 'Doc'
             else:
-                dir = os.path.join('man', 'man1')
+                dir = os.path.join('share', 'man', 'man1')
             self.data_files = [(dir, man_pages)]
             man_dir = os.path.join(self.install_dir, dir)
             msg = "Installed SCons man pages into %s" % man_dir
@@ -402,7 +402,7 @@
                           "SCons.Variables",
                          ],
     'package_dir'      : {'' : 'engine'},
-    'data_files'       : [('man/man1', man_pages)],
+    'data_files'       : [('share/man/man1', man_pages)],
     'scripts'          : scripts,
     'cmdclass'         : {'install'         : install,
                           'install_lib'     : install_lib,
